{{cookiecutter.project_slug}}/clients/mobile/react-native/package.json

```diff
@@ -80,7 +80,7 @@
     "@babel/preset-typescript": "^7.18.6",
     "@expo/config-plugins": "~8.0.0",
     "@react-native-community/eslint-config": "2.0.0",
-    "@tanstack/eslint-plugin-query": "^5.17.20",
+    "@tanstack/eslint-plugin-query": "5.35.6",
     "@types/i18n-js": "^3.8.3",
```

Comment:
@lakardion looks like they release v 5.43.0 this morning which uses a new version of eslint that has breaking changes